src/rgw/driver/rados/rgw_obj_manifest.h

```diff
@@ -471,16 +471,24 @@ class RGWObjManifest {
       return tier_type;
   }
 
+  bool is_tier_type_s3() {
+      return (tier_type == "cloud-s3" || tier_type == "cloud-s3-glacier");
```

Comment:
IMO better to seperate them out

Reply:
We would still need one common routine for all s3-* style tier types as most of the S3 endpoint options and processing would be common. And there is already one defined for just "glacier" below to use when needed specific to glacier